modules/core/src/main/java/com/smartbear/readyapi4j/support/ExecutionLogger.java

```diff
@@ -14,14 +17,16 @@
 import java.util.List;
 import java.util.Map;
 
+import static com.smartbear.readyapi4j.support.RecipeLogger.createFileName;
+
 /**
  * ExecutionListener that writes response HAR entries to a single log file after execution
  */
 public class ExecutionLogger implements ExecutionListener {
 
     private final static Logger LOG = LoggerFactory.getLogger(ExecutionLogger.class);
 
-    public static final String DEFAULT_EXTENSION = "log";
+    public static final String DEFAULT_EXTENSION = "json";
```

Comment:
I choose json to get syntax-highlighting in IDEA... I agree that .har sounds better